caravel/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -1,11 +1,56 @@
 import React from 'react';
 import { Panel } from 'react-bootstrap';
+import TimeSeriesLineChart from './charts/TimeSeriesLineChart';
 
-const ChartContainer = function () {
-  return (
-    <Panel header="Chart title">
-      chart goes here
-    </Panel>
-  );
-};
-export default ChartContainer;
+export default class ChartContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      params: this.getParamsFromUrl(),
+      data: props.viz.data,
+      height: window.innerHeight,
+      label1: 'Label 1',
+    };
+  }
+
+  getParamsFromUrl() {
+    const hash = window.location.search;
```

Comment:
curious if we've thought about using react router for url state management? I used it in dataportal and liked it.